src/Spectre.libGenetic/Individual.cpp

```diff
@@ -29,6 +30,30 @@ namespace Spectre::libGenetic
 Individual::Individual(std::vector<bool> &&binaryData):
     m_BinaryData(binaryData) { }
 
+
+Individual::Individual(size_t size, size_t initialFillup, Seed seed)
```

Comment:
`Individual` should be *data container*, not an *algorithm*. This part should be extracted into a `RandomGenerationFactory` which main purpose will be construction of random generation, as discussed.